src/main/resources/synthea.properties

```diff
@@ -192,7 +192,7 @@ generate.payers.selection_behavior = random
 generate.payers.loss_of_care = false
 
 # Add a FHIR terminology service URL to enable the use of ValueSet URIs within code definitions.
-# generate.terminology_service_url = https://r4.ontoserver.csiro.au/fhir
+#generate.terminology_service_url = https://r4.ontoserver.csiro.au/fhir
```

Comment:
Please don't make formatting-only changes in the PRs. This will just make it harder on the reviewer once we submit this to the forked repo.

Reply:
@hankwallace I will make sure to follow existing formatting.

Reply:
Hi @hankwallace, Yes they are not using eclipse default indentation formatting, they have updated for white-space only with size 2.